### PR TITLE
Change the hamburger button back

### PR DESCRIPTION
**Pull request by Asam237**

### CODE REVIEW

File: frontend/src/constants/navigations/App.navigation.tsx [hunk 1]
```diff
@@ -9,10 +9,12 @@ export const AppNavigation = () => {
   return (
     <BrowserRouter>
       <Switch>
-        <Route exact path="/" component={Home} />
-        <Route exact path="/conseils" component={Conseils} />
-        <Route exact path="/lists" component={List} />
-        <Route exact path="/details/:region" component={Detail} />
+        <Suspense fallback="">
```
Comment (elhmn) [inline frontend/src/constants/navigations/App.navigation.tsx:12]: `Suspence` seems to be an experimental Component, I am wondering if we want to add an alpha feature of a framework right now ? What do you think, is there another wait to achieve the same result without using it ?

**Review by elhmn — COMMENTED** [review]
Looks great, I just left a comment about the `Suspense` component. Let me know what you think

File: frontend/src/constants/navigations/App.navigation.tsx (hunk 1 above)
Comment (Asam237) [reply frontend/src/constants/navigations/App.navigation.tsx:12]: In fact, I had thought of using Suspense in the event that I do not have the data yet, I display a loading. But in this case, I haven't used this anymore, so I remove and do a PR.

File: frontend/src/pages/detail/Detail.page.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
+import React, { Suspense, useState } from "react";
```
Comment (elhmn) [inline frontend/src/pages/detail/Detail.page.tsx:1]: we should maybe remove the `Suspense` import from here as well

**Comment by elhmn** [comment]
@Asam237 let's merge this once we have resolved conflicts 👍

**Pull request merged**